src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -255,14 +243,15 @@ private ConsoleColors GetLogLevelConsoleColors(LogLevel logLevel)
             }
         }
 
-        private string GetScopeInformation()
+        private void GetScopeInformation(StringBuilder builder)
         {
             var current = ConsoleLogScope.Current;
-            var output = new StringBuilder();
             string scopeLog = string.Empty;
+            var length = builder.Length;
```

Comment:
Since the tests also changed, please do a manual verification with scopes on that this behaves the same way